### DISCUSSION TITLE
Scripting

### DISCUSSION THREAD

**Discussion by sundalo50** [sproctor/warlock3#13] [Q&A]
HI there,

I've just started using warlock front end on an old laptop running Linux. So far everything seems to be working fine. My only issue is that I can't seem to figure out how to use any scripts I had from Wrath or how to create any new ones. I have a copy of the xml file from Wrath, but it does not show up as a recognized file. Is there any tutorial out there or perhaps I could get some guidance here. Thanks.

**Comment by sproctor** [comment]
There's currently no way to directly import scripts from Wrayth. You can run the script, but you need to save them to individual files in your scripts directory. The extension should be .wiz, .cmd, or .wsl.
